exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -457,6 +464,9 @@
     .paragraph
       padding: 4px
 
+    .fixed-to-responsive
+      display: inline-block
+
```

Comment:
(this is getting off-topic for this PR - maybe we should continue in https://github.com/learningequality/kolibri-exercise-perseus-plugin/issues/16)